src/python/foglamp/translators/omf_translator.py

```diff
@@ -790,7 +740,43 @@ def omf_types_creation():
     except Exception as e:
         message = _message_list["e000011"].format(e)
 
-        _log.error(message)
+        _logger.exception(message)
+        raise Exception(message)
+
+
+def omf_objects_creation():
+    """Creates all the OMF objects
+
+    Raises:
+        Exception: an error occurred during the OMF's objects creation.
+
+    """
+
+    global _sensor_location
+    global _sensor_id
+    global _measurement_id
+
+    global _type_measurement_id
+    global _type_sensor_id
+
+    try:
+        for sensor_info in _sensor_name_type:
+
+            _sensor_id = sensor_info
+            _measurement_id = "measurement_" + _sensor_id
+
+            tmp_type = _sensor_name_type[_sensor_id]
+
+            _type_sensor_id = "type_sensor_id_" + _type_id + "_" + tmp_type
+            _type_measurement_id = "type_measurement_" + _type_id + "_" + tmp_type
+
+            debug_msg_write("INFO", "OMF_object_creation ")
+            omf_object_creation()
```

Comment:
@sstefano69 I prefer you don't open a new PR, and incorporate changes in this one only. This way we can track what has been changed and incorporate missing functionalities incrementally.